Sources/Plot/API/HTML.swift

```diff
@@ -48,7 +48,7 @@ public extension HTML {
     /// The context within an HTML `<a>` element.
     final class AnchorContext: BodyContext, HTMLLinkableContext {}
     /// The context within an HTML `<audio>` element.
-    enum AudioContext: HTMLMediaContext {
+    enum AudioContext: HTMLMultiMediaContext {
```

Comment:
"Multimedia" is one word, so I think the correct camelCasing here should be: `HTMLMultimediaContext`.

Reply:
Thanks! Will rename.